cp/cp_config.h

```diff
@@ -0,0 +1,62 @@
+/*
+* Copyright 2019-present Open Networking Foundation
+*
+* SPDX-License-Identifier: Apache-2.0
+*
+*/
+#ifndef __CP_CONFIG_H__
+#define __CP_CONFIG_H__
+
+#define DP_SITE_NAME_MAX 128
+#include <stdint.h>
+#include <sys/queue.h>
+
+extern struct app_config *appl_config; 
+
+struct mcc_mnc_key 
+{
+	uint8_t mcc_digit_1 :4;
+	uint8_t mcc_digit_2 :4;
+	uint8_t mcc_digit_3 :4;
+	uint8_t mnc_digit_3 :4;
+	uint8_t mnc_digit_1 :4;
+	uint8_t mnc_digit_2 :4;
+};
+
+
+struct dp_key
+{
+    struct mcc_mnc_key mcc_mnc;
+    struct in_addr enb_addrs[2];
```

Comment:
where will this be used?
why two?

Reply:
I have removed end_addrs for now. I don't want any confusion.